app/uk/gov/hmrc/eoricommoncomponent/frontend/forms/models/ContactDetailsModel.scala

```diff
@@ -45,17 +45,13 @@ case class ContactDetailsModel(
     countryCode.getOrElse("")
   )
 
-  def toContactDetailsViewModel: ContactDetailsViewModel = ContactDetailsViewModel(
-    fullName,
-    Some(emailAddress),
-    telephone,
-    fax,
-    useAddressFromRegistrationDetails,
-    trim(street),
-    trim(city),
-    trim(postcode),
-    countryCode
-  )
+  def toContactsInfoViewModel: ContactDetailsViewModel =
```

Comment:
This should be named as toContactInfoViewModel

Reply:
This has been changed to toContactInfoViewModel, and related references to this has been updated.